build-configs/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.base.json",
+  "extends": "../tsconfig.json",
```

Comment:
Why this change?

Reply:
To enable these rules with typescript support I had to set the parserOptions in .eslintrc.js to a ts config. To allow linting from the root dir as well as from the different workspaces the same name is needed. At least I don't know how to allow that otherwise.